astropy/convolution/convolve.py

```diff
@@ -85,6 +85,10 @@ def _next_fast_lengths(shape):
         return np.array([scipy.fft.next_fast_len(j, real=False) for j in shape])
     except ImportError:
         pass
+    except TypeError as ex:
+        # this error happens with scipy version <=1.5
+        if "good_size() takes no keyword arguments" not in str(ex):
+            raise ex
```

Comment:
```suggestion
        return np.array([scipy.fft.next_fast_len(j) for j in shape])
    except ImportError:
        pass
```
Perhaps try this? `real=False` is the default anyway, and I checked with scipy 1.5 on macos that it seems to work the same way there, but passing either value to the kwarg raises the same exception, although the kwarg is documented there. So either a documentation issue or bug with scipy, which might explain why it did not pop up with 1.1.